src/context/cooperation-context.tsx

```diff
@@ -29,15 +35,35 @@ const CooperationContext = createContext<CooperationProviderContext>(
 const CooperationProvider: FC<CooperationProviderProps> = ({ children }) => {
   const [selectedCourse, setSelectedCourse] = useState<Course | null>(null)
   const [isActivityCreated, setIsActivityCreated] = useState<boolean>(false)
+  const [isCourseAdded, setIsCourseAdded] = useState<boolean>(false)
```

Comment:
didn`t see where you are using this state

Reply:
I agree. 
I've fixed that.